crates/debugger_ui/src/debugger_panel.rs

```diff
@@ -192,7 +192,11 @@ impl DebugPanel {
                 editor.go_to_line::<DebugCurrentRowHighlight>(
                     row,
                     column,
-                    Some(cx.theme().colors().editor_highlighted_line_background),
+                    Some(
+                        cx.theme()
+                            .colors()
+                            .editor_document_highlight_read_background,
```

Comment:
This should be the new color, right? `editor_debugger_active_line_background`

Reply:
fixed it! Thanks for the heads up